release.rst

```diff
@@ -0,0 +1,54 @@
+Instructions for Creating a DAGMC Release
+============================================
+
+Because of a complex dependency between DAGMC and PyNE the following procedures
+are recommended for navigating those dependencies for a new release.
+
+DAGMC depends on PyNE only as a submodule from which C++ files are directly
+incorporated into the build.  It may be desirable to perform a patch-level
+release of DAGMC when any of the C++ files on PyNE change.
+
+PyNE depends on DAGMC as a third-party library that must be installed prior to
+building PyNE.  Changes in DAGMC do not necessarily require updates to PyNE, 
+so may not result in a patch-level release of PyNE.
+
+It is often desirable to perform patch-level releases of PyNE and DAGMC on
+concert.  If so, the following procedure should reduce conflicts in the
+dependencies.
+
+Assumptions
+------------
+
+1. The DAGMC repository has a `stable` tag that points to the same SHA as the newest release.
```

Comment:
```suggestion
1. The DAGMC repository has a `stable` Docker image tag that points to the same SHA as the newest release.
```

Reply:
actually - this is a tag in the source code repo - but perhaps I should still clarify

Reply:
I'm actually not sure how to clarify this without being very wordy.  Maybe we should use something other than `stable` as the tag since this so often is used to specify a docker image

Reply:
Ah, my mistake. `stable` makes sense to me as long as we clarify.